composer.json

```diff
@@ -26,11 +26,12 @@
         "adlawson/vfs": "^0.12.1",
         "zircote/swagger-php": "^3.0",
         "symfony/phpunit-bridge": "4.4.*",
-        "centreon/centreon-test-lib": "dev-master",
+        "centreon/centreon-test-lib": "dev-MON-5020-api-test",
```

Comment:
shouldn't you revert it to master, as your tests succeeded ?

Reply:
I wait to improve centreon-test-lib branch. Once it's done and merge, I'll put back dev-master